x-pack/plugins/ml/public/application/components/custom_urls/custom_url_editor/utils.ts

```diff
@@ -219,7 +291,7 @@ function buildDiscoverUrlFromSettings(settings) {
   });
 
   // Add the index pattern and query to the appState part of the URL.
-  const appState = {
+  const appState: any = {
```

Comment:
did you check if discover exposes a type for `_a`?

Reply:
Updated type in 642ce6dbe70bb719b6416e5a61184a6b966b71d7